R/print_debug.R

```diff
@@ -27,6 +27,12 @@ print_debug <- function(...) {
         # provide helpful debug without breaking the program
         val <- sprintf("ERROR - no %s defined", name)
       }
+    } else if (length(val) > 1) {
+      if (is.character(val)) {
+        val <- sprintf("c('%s')", paste(val, collapse = "', '"))
```

Comment:
Could potentially be confusing if string contains single quotes, but I don't have an immediately obvious way to address that. It's not a big deal anyway.

Reply:
You're absolutely right. This is a *super* barebones implementation. But it's also the least amount of code I needed ...